core/pom.xml

```diff
@@ -81,6 +81,13 @@
             <version>${json.org.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>info.debatty</groupId>
+            <artifactId>java-string-similarity</artifactId>
```

Comment:
Yes sure, if we want to merge this into master. I haven't done this, yet, as I considered the branch to be a bit "experimental - playing around with JaroWinkler".